src/Microsoft.AspNetCore.ResponseCompression/ResponseCompressionExtensions.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Builder;
+using Microsoft.Extensions.Options;
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.AspNetCore.ResponseCompression
+{
+    public static class ResponseCompressionExtensions
+    {
+        /// <summary>
+        /// Allows to compress HTTP Responses.
+        /// </summary>
+        /// <param name="builder">The <see cref="IApplicationBuilder"/> instance this method extends.</param>
+        /// <param name="mimeTypes">The accepted MIME types.</param>
+        public static IApplicationBuilder UseResponseCompression(this IApplicationBuilder builder, IEnumerable<string> mimeTypes)
```

Comment:
I will remove this overload in a first time.

And I was thinking that is won't be easy for a user to add something like "text/*". It will have to add MIME types one by one.
It would be nice to have a `MimeTypeSet` more or less like that:

```
class MimeTypeSet
{
   public void Include(Regex mimeType) { ... }
   public void Include(string mimeType) { ... }
   public void Exclude(Regex mimeType) { ... }
   public void Exclude(string mimeType) { ... }

   public bool Contains(string mimeType) { ... }
}
```
Maybe in the `Microsoft.AspNetCore.WebUtilities` assembly?